README.md

```diff
@@ -48,9 +51,12 @@ compiler, CMake and, optionally, zlib.
 
 ```bash
 $ git clone https://github.com/Equinor/libecl
-$ pip install -r libecl/requirements.txt
-$ cmake libecl -DENABLE_PYTHON=ON
-$ cmake --build libecl --target install
+$ mkdir libecl/build
+$ cd libecl/build
+$ pip install -r ../requirements.txt  # You need write-access to Python's packages
```

Comment:
I think the comment is not necessary. If someone cannot pip install something, the great internet will explain how they can mitigate

Reply:
The issue is that it’s not immediately obvious that pip has failed, and our CMake falls back to disabling Python if it can’t find Python stuff instead of erroring.

Reply:
I agree. But at the same time there are multiple ways pip can fail, perhaps a general warning about checking that the pip installation was successful is better? Or maybe even inspecting the exit code of pip?